boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_m7_ddr_defconfig

```diff
@@ -11,3 +11,4 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_CONSOLE=y
 CONFIG_XIP=y
 CONFIG_CODE_DDR=y
+CONFIG_GPIO=y
```

Comment:
No need to enable GPIO at here, it will be enabled in case of dts nodes is enabled.